content/blog/jjpx/2018-09-05_haiku_monthly_activity_report_august_2018.md

```diff
@@ -0,0 +1,43 @@
++++
+type = "blog"
+title = "Haiku monthly activity report - 08/2018"
+author = "jjpx"
+date = "2018-09-05 16:37:40-03:00"
+tags = []
++++
+
+Time has come and a new monthly activity report arrived with it. It is the first time I write something this long in english. Enjoy!
+
+This report covers hrev52141-52310.
+
+### Drivers
+
+SDHCI MMC driver from [krish_iyer](https://www.haiku-os.org/blog/krish_iyer) has been merged in master. There is still a lot of work to do and it is not enabled in automated builds yet, but should serve as a basis for any future related work. His commits include extensive documentation on how to setup a testing vm, what has been done and what is left to do.
+
+There have been several contributions to the ARM port including work by kallisti5 on our implementation fo Flattened Device Tree (FDT) that should make it easier for Haiku to be run on more ARM hardware variants and a rewrite of the memcpy function for ARM devices by Augustin Cavalier based on Clang 8.
+
+Finally on the driver related news kallisti5 fixed an SMAP violation on s3 accelerant.
+
+### Fixes
+
+PulkoMandy fixed a bug in MediaPlayer related to playing streams of unknown length such as an online radio.
+
+phuodoin fixed a bug where the system became unresponsive when a textsearch is used in a big search.
+
+### Others
+
+waddlesplash fixed and enabled Reiserfs on x86_64 builds. Previously this filesystem was only available on x86 builds.
```

Comment:
Add a bit of background information:

"
Reiserfs is a filesystem originally developed by Hans Reiser for Linux. It was popular for Haiku users because it could be used to build Haiku without needing xattrs emulation, unlike ext3. However, development of Reiserfs has stopped and the focus in the future for Linux filesystems will be on XFS, JFS, or btrfs.